### DISCUSSION TITLE
Divide the central solenoid into two or more concentric sub-coils

### DISCUSSION THREAD

**Discussion by stuartmuldrew** [ukaea/PROCESS#3259] [Ideas]
In issue #410 about the Central Solenoid, @mkovari proposed the idea:  

> There should be an option to divide the coil into two or more concentric sub-coils, with different current densities. The field is lower on the outside of the coil so the superconductor in this region can have a bigger current density.

**Comment by mkovari** [comment]
This is what is usually called a graded coil, and is standard throughout the superconducting magnet industry.
